lib/src/base-styles/_nx-tables.scss

```diff
@@ -9,7 +9,7 @@
   #Tables
 */
 .nx-table {
-  border-collapse: collapse;
+  border-spacing: 0;
```

Comment:
I'm curious about the why of this change.

Reply:
> I'm curious about the why of this change.

IIRC with collapsed borders, the border separating the headers from the body rows would scroll instead of remaining stationary.  So I had to make changes to make it unambiguous that that border is part of the header and not the body.